integrations/docker/build.sh

```diff
@@ -72,6 +72,11 @@ if [[ ${*/--no-cache//} != "${*}" ]]; then
     BUILD_ARGS+=(--no-cache)
 fi
 
+# Prepare context if required
```

Comment:
Please update the comment to describe what the build context is - as it stands the comment of "prepare context is required" does not seem to help me understand the following lines better: I can already read that we do a "check if this shell script exists and if so source it". What is the intent here? Why is it "source" and not execution?